galley/pkg/config/analysis/analyzers/injection/injection-version.go

```diff
@@ -49,12 +51,31 @@ func (a *VersionAnalyzer) Metadata() analysis.Metadata {
 		Inputs: collection.Names{
 			collections.K8SCoreV1Namespaces.Name(),
 			collections.K8SCoreV1Pods.Name(),
+			collections.K8SCoreV1Configmaps.Name(),
 		},
 	}
 }
 
 // Analyze implements Analyzer.
 func (a *VersionAnalyzer) Analyze(c analysis.Context) {
+	var injectionVersion string
+	c.ForEach(collections.K8SCoreV1Configmaps.Name(), func(r *resource.Instance) bool {
```

Comment:
You can use `c.Find(c collection.Name, name resource.FullName)` to look up the resource by name, which makes this more readable

Reply:
Done. thanks!